modules/oadp-creating-restore-cr.adoc

```diff
@@ -14,6 +14,7 @@ You restore a `Backup` custom resource (CR) by creating a `Restore` CR.
 * You must install the OpenShift API for Data Protection (OADP) Operator.
 * The `DataProtectionApplication` CR must be in a `Ready` state.
 * You must have a Velero `Backup` CR.
+* Ensure that PV capacity at backup time matches the requested size, adjusting the requested size to match capacity.
```

Comment:
I would suggest removing the bit after the comma. It seems repetitive of what you have in the first part of the sentence. 
```suggestion
* Ensure that persistent volume (PV) capacity at backup time matches the requested size.
```

Reply:
I raised this issue with the SME who said that "It may not be obvious to average user who maybe conscious of space requested."

Reply:
I think keeping the clause as is still presents confusion. I have played with the wording a bit. what do you think of:
```suggestion
* Ensure to adjust the requested size so the persistent volume (PV) capacity matches the requested size at backup time.
```